geolocation-API/getCurrentPosition_permission_deny.https.html

```diff
@@ -15,20 +15,21 @@
     onSuccess, onError, hasMethodReturned = false;
 
 t.step(function() {
-  onSuccess = t.step_func(function(pos) {
+  onSuccess = t.step_func_done(function(pos) {
     assert_unreached('A success callback was invoked unexpectedly with position ' + positionToString(pos));
   });
 
-  onError =  t.step_func(function(err) {
+  onError =  t.step_func_done(function(err) {
     // http://dev.w3.org/geo/api/test-suite/t.html?00031
     assert_true(hasMethodReturned, 'Check that getCurrentPosition returns synchronously before any callbacks are invoked');
-    assert_equals(err.code, err.PERMISSION_DENIED,
-        'PossitionError code: ' + err.code, + ', message: ' + err.message);
-    done();
+    assert_equals(err.code, err.PERMISSION_DENIED, errorToString(err));
   });
 
+try {
   geo.getCurrentPosition(onSuccess, onError);
   hasMethodReturned = true;
-});
```

Comment:
I don't think I understand this change? We're still inside the t.step() from line 17 here, right? So no need for the try/catch. And since you removed the `});`, this test now seems to be failing with `Uncaught SyntaxError: Unexpected end of input`?